packages/eslint-config-sheriff/src/getAstroConfig.ts

```diff
@@ -1,30 +1,40 @@
 import astro from 'eslint-plugin-astro';
 import astroParser from 'astro-eslint-parser';
+import type { FlatConfig } from '@typescript-eslint/utils/ts-eslint';
 
 export const getAstroConfig = (
   hasReact: boolean,
   customTSConfigPath: string | string[] | undefined,
-) => {
-  return {
-    files: ['**/*.astro'],
-    plugins: {
-      astro,
-    },
-    languageOptions: {
-      parser: astroParser,
-      parserOptions: {
-        parser: '@typescript-eslint/parser',
-        project: customTSConfigPath || true,
-        extraFileExtensions: ['.astro'],
+): FlatConfig.ConfigArray => {
```

Comment:
i think we should remove this. We are okay with the inferred types, plus i already tried to use `@typescript-eslint` types in the past and it didn't end well, because they don't get exported along the rest of the package and end up erroring out in the .dts build.

Reply:
> i think we should remove this. We are okay with the inferred types, plus i already tried to use `@typescript-eslint` types in the past and it didn't end well, because they don't get exported along the rest of the package and end up erroring out in the .dts build.

Huh. I was running into the opposite, the .d.ts build would fail to build _without_ explicitly setting it becuase it wasn't inferred correctly, but for some reason it worked with it.

Reply:
Interesting. I think I'll look at this closely myself then.

Reply:
Oh, to be clear, I don't think it was on this branch, but I recall there being an issue with functions returning configs getting incorrect types inferred.

Reply:
I removed it. Seemed to work fine for now.